.github/workflows/release.yml

```diff
@@ -456,6 +469,23 @@ jobs:
         # the end of this workflow.
         rm -f bundles/*.sha256
 
+    - name: Build package
+      run: |
+        cd build
+
+        # Combine the `openttd` binaries from each build into a single file
+        mv openttd openttd-x64
+        lipo -create -output openttd openttd-x64 ../build-arm64/openttd
```

Comment:
Suggestion:

`
lipo -create -output openttd-universal ../build*/openttd
mv openttd-universal openttd
`

That was it doesn't need to know the targets hard-coded :)

If you change `build` into `build-x64` earlier, make this `build-*` instead of `build*`.